plugins/woocommerce/tests/php/src/Internal/CLI/Migrator/Core/WooCommerceProductImporterTest.php

```
@@ -0,0 +1,809 @@
+		$dry_run_importer->configure( array( 'dry_run' => true ) );
+
+		$product_data           = MockShopifyData::get_mock_wc_product_data( 40 );
+		$product_data['images'] = array(
+			array(
+				'src' => 'https://example.com/image1.jpg',
+				'alt' => 'Test Image',
+			),
+		);
+
+		// In dry run mode, import should still work but not create actual products.
+		$result = $dry_run_importer->import_product( $product_data );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (Unused Code Rules)

(UnusedLocalVariable)